### ISSUE TITLE
[tableio] example scripts are broken (Trac #1259)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1253]
Example script `first_light.py` is broken:

- `SubEventStreams` parameter is not set
- root: python arguments do not match the C++ signature
- csv: `textwriter` module does not exist anymore

Migrated from https://code.icecube.wisc.edu/ticket/1259
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:13:59", 
    "description": "Example script `first_light.py` is broken:\n\n- `SubEventStreams` parameter is not set\n- root: python arguments do not match the C++ signature\n- csv: `textwriter` module does not exist anymore", 
    "reporter": "kkrings", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335639558230", 
    "component": "combo core", 
    "summary": "[tableio] example scripts are broken", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-20T20:24:04", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T21:03:11: 
* kkrings changed summary from "[tableio] first_light is broken" to "[tableio] example scripts are broken"
* kkrings commented:

> `merge.py` and `transcribe.py` are also broken.

**Comment by icecube-trac** [comment]
Trac update at 20150824T17:54:04: olivas changed milestone from "offline V15-09-00" to "offline V15-08-00"

**Comment by icecube-trac** [comment]
Trac update at 20150826T09:56:25: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 136878:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136878"
> Fix #1253 with the fixing hammer.
> ```


* jvansanten changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas commented:

> Milestone offline V15-08-00 deleted


* olivas changed milestone from "offline V15-08-00" to ""
